packages/indexer-service/src/commands/start.ts

```diff
@@ -36,7 +38,9 @@ export default {
   describe: 'Start the service',
   builder: (yargs: Argv): Argv => {
     return yargs
-      .option('ethereum', {
+      .option('network-provider', {
+        //TODO:FIXME: uptrade this field as we did for Agent
```

Comment:
?

Reply:
I think that comment is outdated. It was made when we were considering multiplying the startup arguments at the CLI level, but we decided to switch to network specification files instead. So, that comment is no longer relevant.


Removed it in 6cfbfd57